packages/stark-ui/src/modules/date-range-picker/components/date-range-picker.component.ts

```diff
@@ -0,0 +1,143 @@
+import { Component, EventEmitter, Input, Output, ViewChild, ViewEncapsulation } from "@angular/core";
+import { StarkDatePickerFilter, StarkDatePickerComponent } from "./../../date-picker/components/date-picker.component";
+import moment from "moment";
+
+/**
+ * StarkDateRangePickerEvent interface
+ */
+export interface StarkDateRangePickerEvent {
+	startDate: Date | undefined;
+	endDate: Date | undefined;
+}
+
+/**
+ * Component to display the stark date-range-picker
+ */
+@Component({
+	selector: "stark-date-range-picker",
+	templateUrl: "./date-range-picker.component.html",
+	encapsulation: ViewEncapsulation.None
+})
+export class StarkDateRangePickerComponent {
+	/**
+	 * Filter function or a string
+	 * Will be applied to both date-picker
+	 */
+	@Input()
+	public dateFilter?: StarkDatePickerFilter;
+
+	/**
+	 * Source Date to be bound to the end datepicker model
+	 */
+	@Input()
+	public endDate: Date | undefined;
+
+	/**
+	 * Label to be displayed in the end datepicker
+	 */
+	@Input()
+	public endDateLabel: string;
+
+	/**
+	 * Maximum date of the end date picker
+	 */
+	@Input()
+	public endMaxDate: Date;
+
+	/**
+	 * Minimum date of the end date picker
+	 */
+	@Input()
+	public endMinDate: Date;
+
+	/**
+	 * Whether the datepickers are disabled
+	 */
+	@Input()
+	public isDisabled?: boolean;
+
+	/**
+	 * HTML "name" attribute of the element.
+	 */
+	@Input()
+	public rangePickerId: string = "";
+
+	/**
+	 * HTML "name" attribute of the element.
+	 */
+	@Input()
+	public rangePickerName: string = "";
+
+	/**
+	 * Source Date to be bound to the start datepicker model
+	 */
+
+	@Input()
+	public startDate: Date | undefined;
+
+	/**
+	 * Label to be displayed in the start datepicker
+	 */
+	@Input()
+	public startDateLabel: string;
```

Comment:
Could you please provide a default key value? 
With "From" / "De" / "Van" as translated values